opds_import.py

```diff
@@ -925,50 +879,143 @@ def __init__(self, _db, feed_url, default_data_source, import_class,
             keep_timestamp=keep_timestamp, default_start_time=Monitor.NEVER
         )
     
-    def _get(self, url):
+    def _get(self, url, headers):
         """Make the sort of HTTP request that's normal for an OPDS feed.
 
         Long timeout, raise error on anything but 2xx or 3xx.
         """
         kwargs = dict(timeout=120, allowed_response_codes=['2xx', '3xx'])
-        return HTTP.get_with_timeout(url, **kwargs)
+        response = HTTP.get_with_timeout(url, headers=headers, **kwargs)
+        return response.status_code, response.content_type, response.content
```

Comment:
We only need the content, but this matches the DummyHTTPClient so it's easier to test.